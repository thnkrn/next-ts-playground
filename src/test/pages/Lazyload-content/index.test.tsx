import LazyloadContent from 'pages/Lazyload-content'
import { render } from 'test/util'

describe('LazyloadContent', () => {
  describe('Rendering', () => {
    it('should match snapshot', () => {
      const { asFragment } = render(<LazyloadContent />)

      expect(asFragment()).toMatchSnapshot()
    })
  })
})
