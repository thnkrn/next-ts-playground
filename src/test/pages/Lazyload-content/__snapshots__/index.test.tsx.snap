// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LazyloadContent Rendering should match snapshot 1`] = `
<DocumentFragment>
  <style
    data-cmpld="true"
  >
    ._19pk1wug{margin-top:auto}._2hwx1wug{margin-right:auto}._otyr1wug{margin-bottom:auto}._18u01wug{margin-left:auto}._1bsb13sl{width:85%}
  </style>
  <div
    class="_19pk1wug _2hwx1wug _otyr1wug _18u01wug _1bsb13sl"
  >
    <h1
      class="h1"
    >
      Lazyload content suspense
    </h1>
    <button>
      Show content
    </button>
  </div>
</DocumentFragment>
`;
